### DISCUSSION TITLE
Sonoff SNZB-06P sensitivity

### DISCUSSION THREAD

**Discussion by madner81** [Koenkk/zigbee2mqtt#20070] [Devices]
The eWeLink app allows to change the sensitivity of the SNZB-06P presence sensor. Is it possible to introduce this device-specific setting? Thank you.

**Comment by somebody-somewhere-over-the-rainbow** [comment]
The device should also support illumination level or at least daylight vs night brightness. So far that information is not available in Z2M

**Comment by sorryusernameisalreadytaken** [comment]
> The eWeLink app allows to change the sensitivity of the SNZB-06P presence sensor. Is it possible to introduce this device-specific setting? Thank you.

You can set this, see here:
https://github.com/Koenkk/zigbee2mqtt/issues/19771#issuecomment-1823794637

Same with "ultrasonicOToUDelay", but this should be the timeout. The dafult value was 60 (seconds) here on my device.
